src/main/java/Task.java

```diff
@@ -0,0 +1,36 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        if (isDone) { // mark done task with X
```

Comment:
Avoid putting the comment on same line as this statement. It should be placed a line above, be indented relative to their position of the code statement if(isDone).